components/home/menu/calendar/Screens/DashboardScreen/index.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableOpacity, StyleSheet, Alert } from 'react-native';
+import { Agenda } from 'react-native-calendars';
+import styles from './styles';
+
+export default class DashboardScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: {},
+      synchronizedEvents: this.structureSynchronizedEvents(
+        props.navigation.state.params.jsonFile.items
+      )
+    };
+  }
+
+  structureSynchronizedEvents(events) {
+    let tempArray = [];
+    events.forEach(event => {
+      tempArray.push(
+        {
+          date: event.start.dateTime != null ? event.start.dateTime.substring(0,event.start.dateTime.indexOf('T')):event.start.date,
```

Comment:
Spacing inconsistencies. Should use the linter to fix the code quality of all the changed files.

Reply:
Linter has been fixed